float64 timestamp
float64 ch1_K

